public/index.html

```diff
@@ -1,6 +1,19 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=%REACT_APP_MEASUREMENT_ID%"></script>
+    <script>
+      window.dataLayer = window.dataLayer || [];
+      function gtag(){dataLayer.push(arguments);}
+      gtag('js', new Date());
+      gtag('consent', 'default', {
+        'ad_storage': 'denied',
```

Comment:
That link describes disabling cookies using Analytics.js which is an older version of the newer gtag.js library (some comparisons are described in this blog post: https://www.analyticsmania.com/post/gtag-vs-analytics-js/). I used the gtag.js library since it is newer and Analytics.js is no longer updated.

That being said I may try to switch over to the Analytics.js library snippet since I am having trouble seeing the tracking activity on the Google Analytics console while the `analytics_storage` property is set to `denied`. There also seems to be a lot less documentation on the newer gtag.js library, especially when it comes to disabling cookies.

Reply:
I tested switching over to the Analytics.js script as suggested (using the storage: 'none' config option), and the functionality remained the same (see [google-analytics-ua-alternative branch](https://github.com/dvci/shc-web-verifier/blob/google-analytics-ua-alternative/public/index.html#L9) where I tested this alternative approach).

On this branch I was also able to originally see website traffic locally, but when I disabled cookies, I was no longer able to see new active users on the analytics console. 

I presume this may start working again once the code is deployed to https://dvci.github.io/shc-web-verifier, instead of being run locally.